py3status/modules/net_iplist.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+"""
+Display the list of current IPs.
+
+This can exclude specified IPs and displays "no connection" if there are no IPs
+to display.
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds.
+        (default 30)
+    format: format of the output.
+        (default 'Network: {format_iface}')
+    format_iface: format string for the list of IPs of each interface.
+        (default '{iface}: v4{{{ip}}} v6{{{ip6}}}')
+    iface_blacklist: list of interfaces to ignore.
+        (default ['lo'])
+    iface_sep: string to write between interfaces.
+        (default ' ')
+    ip_blacklist: list of IPs to ignore. Accepts shell-style wildcards.
```

Comment:
an example would be good, either here or as an example at the end (see https://github.com/ultrabug/py3status/blob/master/doc/README.md#docstring)